dockernet/scripts/community-pool-staking/stake_proposal.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+set -eu 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/../../config.sh
+
+deposit_ica_account=$(GET_ICA_ADDR DYDX community_pool_deposit)
+proposal_file=${STATE}/${DYDX_NODE_PREFIX}1/pool.json
+cat << EOF > $proposal_file
+{
+  "title": "Community Spend: Liquid stake",
+  "metadata": "Community Spend: Liquid stake",
+  "summary": "Community Spend: Liquid stake",
+  "messages": [
+    {
+      "@type": "/cosmos.distribution.v1beta1.MsgCommunityPoolSpend",
+      "authority": "dydx10d07y265gmmuvt4z0w9aw880jnsr700jnmapky",
+      "recipient": "$deposit_ica_account",
+      "amount": [
+        {
+            "amount": "1000",
+            "denom": "udydx"
+        }
+      ]
+    }
+  ],
+  "deposit": "2000000000udydx"
+}
+EOF
+
+echo ">>> Submitting proposal to spend community pool tokens..."
+$DYDX_MAIN_CMD tx gov submit-proposal $proposal_file --from ${DYDX_VAL_PREFIX}1 -y | TRIM_TX
```

Comment:
Nice, this is a good test for phase 1/3 since this is how we know that community pool action will actually originate.  Is it the same type of gov proposal for each of the community pools (since we now know there could be more than one) or are some of them just module accounts which don't require the gov actions to kick them off?

Reply:
my understanding is the other "community pools" are just normal module accounts and they can do a gov prop to do a bank send - but it's functionally the same as a normal bank send